features/step_definitions/metering.rb

```diff
@@ -225,3 +224,47 @@
   res = host.exec(shell_cmd)
   raise "#{cb.metering_namespace} #{op} unsuccessfully" unless res[:success]
 end
+
+# install metering via OLM,
+Given /^the#{OPT_QUOTED} metering service is installed(?: to $QUOTED)? using OLM$/ do | metering_ns |
+  ensure_admin_tagged
+  # 1. create the metering namespace
+  metering_ns ||= "openshift-metering"
+  project(metering_ns)
+  step %Q/I switch to cluster admin pseudo user/
+  @result = user.cli_exec(:create_namespace, name: metering_ns)
```

Comment:
you don't need to switch to cluster admin. You can do `admin.cli_exec(...)`.

Reply:
@akostadinov, but for step on line 250, the user needs to be admin, so I think we can just do it here.